### PR TITLE
Modified Student/Route Table

### LINKED ISSUE #266: Backend: Student/Route Table modification
Add is_range boolean to Student Table and add to Route table color_id, fix existing api's to handle this new change

### PR DESCRIPTION
**Pull request by eve658**
Closes #266 

Note: Student create will also break because you need to send the in_range boolean. Example JSON's were updated to reflect these table changes.

Once this PR is merged into dev, run the following command in the backend:
`python manage.py migrate`

### CODE REVIEW

File: backend/ht_buses_app/views/routes/route_create.py [hunk 1]
```diff
@@ -21,6 +21,8 @@ def route_create(request):
         description = reqBody["route"]["description"]
         is_complete = reqBody["route"]["is_complete"]
         route = Route.routeTables.create(name=name, school_id = school, description = description, is_complete=is_complete)
+        route.color_id = route.id % 51
```
Comment (ThomasQuesadilla) [inline backend/ht_buses_app/views/routes/route_create.py:24]: might come in handy to move the 51 to a constant variable when you get the chance.

**Review by ThomasQuesadilla — APPROVED** [review]
LGTM

File: backend/ht_buses_app/views/routes/route_create.py (hunk 1 above)
Comment (eve658) [reply backend/ht_buses_app/views/routes/route_create.py:24]: lemme go do that :)

**Review by kyratichan — APPROVED** [review]
lgtm

**Pull request merged**